src/app/[locale]/(pages)/accedi/errore/page.tsx

```diff
@@ -188,13 +188,7 @@ const LoginErrorPage = () => {
               {renderErrorMessage(errorCode).message}
             </Typography>
           </Grid>
-          {typeof errorCode === 'string' && errorCode === '1001' ? (
-            <Grid item xs={12} textAlign={'center'}>
-              <Button onClick={() => pushWithLocale(ROUTES.LOGIN)} variant="contained">
-                {t('common.backtohome')}
-              </Button>
-            </Grid>
-          ) : (
+          {
```

Comment:
can you delete this typo?

Reply:
at line 191 you open a bracket and at line 221 you close it. This bracket is not useful, so we can delete it! 😁